### PR TITLE
[zh]Sync /docs/tasks/traffic-management/ingress/ingress-control/index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/ingress/ingress-control/index.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ $ minikube tunnel
 
 {{< /tip >}}
 
+{{< tip >}}
+如果您正在 Linux 系统上使用 Kind 集群运行 Istio，请、遵循[设置  MetalLB 指南]来获得类型为 `LoadBalancer` 的服务。
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/ingress/ingress-control/index.md:49]: 这是个啥，提交以后，自己一定要 review 读一遍！

Comment (yanrongshi) [reply content/zh/docs/tasks/traffic-management/ingress/ingress-control/index.md:49]: 好的

File: content/zh/docs/tasks/traffic-management/ingress/ingress-control/index.md [hunk 2]
```diff
@@ -45,6 +45,10 @@ $ minikube tunnel
 
 {{< /tip >}}
 
+{{< tip >}}
+如果您正在 Linux 系统上使用 Kind 集群运行 Istio，请遵循[设置 MetalLB 指南]来获得类型为 `LoadBalancer` 的服务。
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/ingress/ingress-control/index.md:49]: Hey, could I ask where is the url of link? Do you know what is review?

Comment (windsonsea) [reply content/zh/docs/tasks/traffic-management/ingress/ingress-control/index.md:49]: 这么小的修改，在反复要求检查之后，还是出现这么低级的错误，很难理解你这 1 年多的 Istio 经验体现在哪里？？？

**Review by windsonsea — CHANGES_REQUESTED** [review]
请认真对待每一件事。
Don't show your careless everywhere. That's a bad and polluted impression.

**Review by windsonsea — CHANGES_REQUESTED** [review]
1. Line 56 and 80 may also need to be localized.
   {{< tab name="external load balancer" category-value="external-lb" >}}
   {{< tab name="node port" category-value="node-port" >}}
2. If you can open and read [this page](https://istio.io/latest/zh/docs/tasks/traffic-management/ingress/ingress-control/), it's easy to find more:

image

> Note: It's very easy to find if you are careful or careless in any PR you opened, which will show your personality to everyone.

**Comment by yanrongshi** [comment]
Wait for the upstream to repair

**Review by zirain — CHANGES_REQUESTED** [review]
same as https://github.com/istio/istio.io/pull/11974？

**Pull request closed without merging**